cmd/kubeturbo/app/kubeturbo_builder.go

```diff
@@ -36,6 +36,8 @@ const (
 	defaultVMPriority           = -1
 	defaultVMIsBase             = true
 	defaultDiscoveryIntervalSec = 600
+	defaultValidationWorkers    = 10
+	defaultValidationTimeout    = 60
```

Comment:
How is the timeout related to the timeout at server side? Seems server has a timeout 30 seconds.

Reply:
The timeout can be configured. These are just defaults. The server will timeout if something is horribly wrong, and server timeouts may change in the future.